docs/deployment/application-deployment.md

```diff
@@ -182,27 +182,27 @@ git push dokku master
 
 ## Dokku/Docker container management compatibility
 
-Dokku is, at its core, a Docker container manager. Thus, it does not necessarily play well with other out-of-band processes interacting with the Docker daemon. One thing to note as in [issue #1220](https://github.com/dokku/dokku/issues/1220), Dokku executes a cleanup function prior to every deployment.
+Dokku is, at its core, a Docker container manager. Thus, it does not necessarily play well with other out-of-band processes interacting with the Docker daemon.
 
-As of 0.5.x, this function removes all containers with the label `dokku` where the status is either `dead` or `exited`, as well as all `dangling` images. Previous versions would remove `dead` or `exited` containers, regardless of their label.
+One thing to be aware of (see [issue #1220](https://github.com/dokku/dokku/issues/1220)) is that Dokku executes a cleanup function prior to every deployment. As of 0.5.x, the cleanup removes all containers with the `dokku` label where the status is either `dead` or `exited` (previous versions would remove _all_ `dead` or `exited` containers). The cleanup function also removes all images with `dangling` status.
 
 ## Adding deploy users
 
 See the [user management documentation](/docs/deployment/user-management.md).
 
-## Default vhost
+## Setting a default vhost
```

Comment:
Mind reverting all these headers? They are essentially here as redirects for users who would have otherwise linked to the header.